### ISSUE TITLE
Use new moonraker config format

### ISSUE THREAD

**Issue by Luro02** [Luro02/klipper-mpc#17]
There seems to be some changes, see https://moonraker.readthedocs.io/en/latest/configuration/#git-repo-configuration

**Comment by chrisheib** [comment]
Thats probably what I encountered too in #16

**Comment by Luro02** [comment]
Nope, those are unrelated. Not sure what is causing the crash, I will investigate this, when I have time to continue working on this port. (Even if that were fixed as mentioned in https://klipper.discourse.group/t/model-predictive-temperature-control-port-for-klipper/12850?u=luro02 you will likely be stuck with a bad calibration)
